i18n/src/main/resources/openfire_i18n.properties

```diff
@@ -3283,9 +3254,9 @@ plugin.available.manual.update = Update Now.
 plugin.available.no.list = The list of available plugins has not yet been downloaded.
 plugin.available.no.plugin = Plugin information has not yet been downloaded.
 plugin.available.list = Click here to download.
-plugin.available.no.list.description =  In order to install plugins, the list of available plugins must be download \
+plugin.available.no.list.description =  In order to install plugins, the list of available plugins must be downloaded \
                     from Jive Software. Once the plugins have been downloaded, you may \
-                    chooser which plugins to install.
+                    choose which plugins to install.
```

Comment:
Is this string even accurate? I don't think we download anything from Jive Software. @guusdk ?

Reply:
Yeah, Jive Software doesn't offer any Openfire support anymore. That can be taken out.